src/main/drivers/motor.c

```diff
@@ -138,6 +138,14 @@ static void analogInitEndpoints(const motorConfig_t *motorConfig, float outputLi
     }
 }
 
+bool isProtocolBidirectionalDshot(const motorDevConfig_t *motorDevConfig)
+{
+    bool motorProtocolDshot = false;
+    const bool motorProtocolEnabled = checkMotorProtocolEnabled(motorDevConfig, &motorProtocolDshot);
+    const bool isBidir = motorDevConfig->useDshotTelemetry;
+    return motorProtocolEnabled && motorProtocolDshot && isBidir;
+}
+
```

Comment:
Could you maybe use the extern `useDshotTelemetry` ?

Reply:
I think you can have DShot telemetry activated in CLI regardless of the motor protocol. I'll check tomorrow to make sure.

Reply:
See comment above... could `motorConfigMutable()->dev.useDshotTelemetry` be suitable?
I can't see where it is ever set to true, though.  If it was defaulted to false, in config.c, then it would only become true when both DShot and DShot telemetry was enabled, later on in config.c.
Would that work?